etc/security-logo-color-64px.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
did you test the rendering of this image?

Reply:
copied it straight out of the endpoint-package repo